buchschloss/config/validation.py

```diff
@@ -7,6 +7,8 @@
 
 from configobj import validate
 
+MAX_LEVEL = 10
```

Comment:
is there a better place for this/

Reply:
Not really. It's the only configuration constant used outside the ``config`` package. While I could also move constants from ``config.main`` to a new file along with this, ``config.main`` is the only place they'll be imported.